charts/lab-partner/values.yaml

```diff
@@ -7,6 +7,9 @@ replicaCount: 1
 mongodb:
   service:
     name: lab-partner-mongodb
+  auth:
+    rootPassword: ""
```

Comment:
left this comment on the other PR, but don't use the root password to connect to mongo.  let's use the `auth.username` and `auth.password` values for the chart to use a regular user that doesn't have elevated credentials.